perfkitbenchmarker/benchmarks/block_storage_database_benchmark.py

```diff
@@ -0,0 +1,128 @@
+# Copyright 2014 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Runs fio benchmarks to simulate database.
+
+Man: http://manpages.ubuntu.com/manpages/natty/man1/fio.1.html
+Quick howto: http://www.bluestop.org/fio/HOWTO.txt
+
+Simulated database benchmark does the following things (PD, PD-SSD, local SSD):
+1) 4K Random R on a file using queue depths 16 and 64 (each queue depth
+   is a different benchmark).
+2) 4K Random W on a file using queue depths 16 and 64 (each queue depth
+   is a different benchmark).
+3) 4K Random 90% R/ 10% W on a file using queue depths 16 and 64 (each
+   queue depth is a different benchmark).
+4) The size of the test file is decided by the test VM's total memory and capped
+   at 1GB to ensure this test finishes within reasonable time.
+
+For AWS, where use PD, we should use EBS-GP and EBS Magnetic, for PD-SSD use
+EBS-GP and PIOPS.
+"""
+
+import json
+import logging
+
+from perfkitbenchmarker import flags
+from perfkitbenchmarker.packages import fio
+
+FLAGS = flags.FLAGS
+
+BENCHMARK_INFO = {'name': 'block_storage_database',
+                  'description': 'Runs FIO to simulate database scenarios.',
```

Comment:
It would be nice to have a very brief description of the scenarios here.

Nit: `fio` should be lowercase throughout.